hazelcast/proxy/topic.py

```diff
@@ -107,6 +124,9 @@ def publish(  # type: ignore[override]
     ) -> None:
         return self._wrapped.publish(message).result()
 
+    def publish_all(self, messages: typing.Sequence[MessageType]) -> None:  # type: ignore[override]
+        return self._wrapped.publish_all(messages).result()
```

Comment:
Can you follow the styling of other definitions and put the ignore thingy into just the method name

Reply:
Fixed at https://github.com/hazelcast/hazelcast-python-client/pull/570/commits/d2d7dc0387ac8e704d4f74533cc1f9d24854773e.

Reply:
Un-resolving this because this is edited again. ı think it was due to the black. If you want to preserve this style even by using the black, add a trailing comma to the last parameter, similar to others

Reply:
Thanks for reminding this, it was due to black. Now fixed at https://github.com/hazelcast/hazelcast-python-client/pull/570/commits/b91a6e5018e63f4a14938e3680d9762d4586863e.